_posts/2018-12-20-downloader.md

```diff
@@ -0,0 +1,115 @@
+a command-line OMERO client. It writes selected data from an OMERO
+server into a local directory and creates soft links to represent some
+of the relationships among server objects. This is still an early
+version missing many features but it can already download some original
+files and metadata.
+
+OMERO.downloader is designed to handle situations in which not all the
+specified data can be downloaded in a single session. If download is
+interrupted then it can be resumed by repeating the same command line
+invocation. If files have already been downloaded then they will not be
+fetched again.
+
+
+# Downloading original files
+
+The files that were uploaded for OMERO image ID 1234 are available
+through:
+
+    ./download.sh -u my-user -w my-pass -s my.omero.server -f binary,companion Image:1234
+
+These are downloaded within the current directory. The `-b` option can
+be used to specify a different preexisting directory to use as a base
+for the downloads. We recommend using a different base directory for
+each OMERO server that you use because the directory structure created
+locally reflects how the server stores your data.
+
+The above command would download image files into the
+`Image/1234/Binary/` directory with any companion files (not containing
+pixel data) in the `Image/1234/Companion/` directory. The files are soft
+links that, perhaps via a `Fileset/` directory, link to files in
+`Repository/`. In the repository directory the binary and companion
+files are located together. On systems with the GNU Core Utilities
+installed a command like:
+
+    showinf `realpath Image/1234/Binary/my-image.fmt`
+
+can be used to conveniently direct Bio-Formats' command-line tools to
+the directory that includes the binary and companion files together.
+
+The original files for multiple images can be downloaded by specifying,
+e.g., `Dataset:123` or `Image:1234,1235,1236`. However, nothing stored
+in the base directory indicates which datasets or other containers held
+the downloaded images. Original files from plates may be downloaded only
+if the server's `omero.policy.binary_access` setting is configured to
+permit it.
+
+
+# Exporting metadata as OME-XML
+
+Metadata representing images, ROIs and some annotations can be fetched
+from the OMERO server and written locally as OME-XML:
+
+    ./download.sh -u my-user -w my-pass -s my.omero.server -f ome-xml Image:1234
+
+The OME-XML is stored in two forms: First, each top-level schema object
+is stored independently in separate files, e.g., in
+`Image/1234/Metadata/image-1234.ome.xml`. Soft links exist among related
+model objects, e.g., `Image/1234/Annotation/567` may link to
+`Annotation/567/` which contains `Metadata/annotation-567.ome.xml`. To
+use those files and links to list the IDs of the images that are tagged
+as "anaphase":
+
+    grep -lr '^<Tag.*<Value>anaphase<' Image/*/Annotation/*/Metadata/ | cut -f 2 -d / | sort -nu
+
```

Comment:
end quotes is missing
The following text now appears in italic

Reply:
It renders fine in Jekyll, I think GH just doesn't understand this variety of markup.